src/pkg/notifier/handler/notification/slack_handler_test.go

```diff
@@ -105,3 +104,32 @@ func TestSlackHandler_Name(t *testing.T) {
 	handler := &SlackHandler{}
 	assert.Equal(t, "Slack", handler.Name())
 }
+
+func Test_escapeEventData(t *testing.T) {
+	type args struct {
+		str string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: `escape "`,
+			args: args{str: `{"foo":"bar"}`},
+			want: `{\"foo\":\"bar\"}`,
+		},
+		{
+			name: `escape \\"`,
+			args: args{str: `{\"foo\":\"bar\"}`},
+			want: `{\\\"foo\\\":\\\"bar\\\"}`,
+		},
```

Comment:
according to the `args` & `want`, it seems that the code is escaping `\"`, instead of `\\"`. Could you please elaborate it? Thanks

Reply:
If the input is `{\"foo\":\"bar\"}`, then `escapeEventData` function will firstly escape the symbol `"`, so the input now will look like `{\\"foo\\":\\"bar\\"}`, but it's not valid because `\` also need to be escaped, so 
 `escapeEventData` function again convert `\\"` => `\\\"` to escape the `\`.

Reply:
sorry, it seems that I understand it now. `{\"foo\":\"bar\"}` is replaced to `{\\"foo\\":\\"bar\\"}` first by ```
	// escape " to \"
	str = strings.Replace(str, `"`, `\"`, -1)
```
and then replaced to `{\\\"foo\\\":\\\"bar\\\"}` by 
```
	// escape \\" to \\\"
	str = strings.Replace(str, `\\"`, `\\\"`, -1)
```